config/config.go

```diff
@@ -0,0 +1,138 @@
+package config
+
+import (
+	"fmt"
+	"github.com/Portshift/klar/docker"
+	"github.com/Portshift/klar/utils"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+//Used to represent the structure of the whitelist YAML file
+type vulnerabilitiesWhitelistYAML struct {
```

Comment:
not used

Reply:
This is part of the current klar (to be used from outside to print). I just moved the file